tests/Dialect/TensorExt/Transforms/implement_shift_network_example.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: heir-opt --implement-shift-network=ciphertext-size=16 %s | FileCheck %s
```

Comment:
what's being checked here beyond validating the pass runs?

Reply:
Nothing. This was the example I put in the documentation, so I felt it should run, but I didn't have the steam to add a detailed list of assertions like the other test.